mage_ai/data_preparation/templates/data_exporters/cratedb.py

```diff
@@ -0,0 +1,31 @@
+from mage_ai.settings.repo import get_repo_path
+from mage_ai.io.config import ConfigFileLoader
+from mage_ai.io.cratedb import CrateDB
+from pandas import DataFrame
+from os import path
+
+if 'data_exporter' not in globals():
+    from mage_ai.data_preparation.decorators import data_exporter
+
+
+@data_exporter
+def export_data_to_cratedb(df: DataFrame, **kwargs) -> None:
+    """
+    Template for exporting data to a PostgreSQL database.
+    Specify your configuration settings in 'io_config.yaml'.
+
+    Docs: https://docs.mage.ai/design/data-loading#postgresql
+    """
```

Comment:
Docstrings need also adjustments, but sure all that can happen after merging to `cratedb`, just any time before submitting to upstream. It will probably take another while. I can also support on such details.